pages/golfers/[userId].js

```diff
@@ -0,0 +1,47 @@
+import { useState, useEffect } from 'react'
+import useUsersScore from '../../lib/useUsersScore'
+import Layout from '../../components/Layout'
+import { getUserId, getUsername } from '../../lib/userAuth'
+
+function GolfersDetails() {
+
+  const [ userName, setUsername ] = useState('')
+  const [ userId, setUserId ] = useState(getUserId())
+  const { scores } = useUsersScore(userId)
```

Comment:
Does this fetch the **current** user's scores despite which golfer's page we're on? If I'm the user with ID 1 and I go to page `/golfers/2` would the page show my scores or their scores (golfer 2's)?

Reply:
With the previous code, an error was thrown. I've changed it back (to set de user id on mount), and also added a condition to useSWR to do not fetch if the user id is not defined. Now, when you try to go to page `/golfers/2`, the page will display the current logged in user details.